android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">10.0.2.2</domain>
+        <domain includeSubdomains="true">localhost</domain>
+    </domain-config>
+</network-security-config>
```

Comment:
This is something that in the long run would be nice to get rid of. Any ideas how we do not forget about this?

Reply:
If I understand it correct even if we do the api with https, we still need these for the tests.